internal/app/machined/pkg/controllers/network/etcfile_test.go

```diff
@@ -223,23 +223,27 @@ func (suite *EtcFileConfigSuite) testFiles(resources []resource.Resource, conten
 }
 
 func (suite *EtcFileConfigSuite) TestComplete() {
+	suite.resolverStatus.TypedSpec().SearchDomains = []string{"foo.example.com"}
+
 	suite.testFiles(
 		[]resource.Resource{suite.cfg, suite.defaultAddress, suite.hostnameStatus, suite.resolverStatus, suite.hostDNSConfig},
```

Comment:
I guess `hostnameStatus` should be dropped from the test

Reply:
@smira Nope, I can't even drop `HostnameStatuses.net.talos.dev` resource input from `network.EtcFileController` since it's used in `network.(*EtcFileController).renderHosts` method.